packages/node/src/evm/authorization/authorization-fetching.test.ts

```diff
@@ -15,12 +15,13 @@ import { RequestStatus } from '../../types';
 import { AirnodeRrp } from '../contracts';
 
 describe('fetch (authorizations)', () => {
-  let mutableFetchOptions: any;
+  let mutableFetchOptions: authorization.FetchOptions;
 
   beforeEach(() => {
     mutableFetchOptions = {
-      address: '0xD5659F26A72A8D718d1955C42B3AE418edB001e0',
-      airnodeId: '0xf5ad700af68118777f79fd1d1c8568f7377d4ae9e9ccce5970fe63bc7a1c1d6d',
+      authorizers: ['0x711c93B32c0D28a5d18feD87434cce11C3e5699B', '0x9E0e23766b0ed0C492804872c5164E9187fB56f5'],
+      airnodeAddress: '0xf5ad700af68118777f79fd1d1c8568f7377d4ae9e9ccce5970fe63bc7a1c1d6d',
```

Comment:
Here I should have used an actual address instead of keeping the same bytes32 id. This is a mistake made by me and I'll fix it on another branch.

Reply:
Ok. I'll resolve this conversation. You can apply the fix after this one is merged.